c2corg_ui/static/js/documentdetailsview.js

```diff
@@ -74,7 +76,13 @@ app.module.directive('appViewDetails', app.viewDetailsDirective);
  * @export
  * @ngInject
  */
-app.ViewDetailsController = function($uibModal, $compile, $scope, appApi) {
+app.ViewDetailsController = function($uibModal, $compile, $scope, appApi, appDocument) {
+
+  /**
+   * @type {app.Document}
+   * @export
+   */
+  this.documentService = appDocument;
```

Comment:
It is exported only in the details-view-controller so I can access it through the template, where I assign the document values to the document service.